libzeth/tests/snarks/groth16/groth16_snark_test.cpp

```diff
@@ -81,7 +81,7 @@ template<typename ppT> void generate_test_data()
 
     // Proving key
     const typename groth16::proving_key pk(
-        G1::one(),                // alpha_g1 = [1]_1
+        G1(G1::one()),            // alpha_g1 = [1]_1
```

Comment:
Why do you need to surround `G1:one()` here?

Reply:
This is required because the constructor of this type (`libsnark::r1cs_gg_ppzksnark_proving_key`) only accepts rvalue refs, but `G1::one` now returns a const ref instead of a value.

(AFAICT there is no advantage to rval refs in that constructor, but I didn't want to start chaning the constructor signatures at this point)

Reply:
Ah, I see. Thanks.